src/common-ui/components/IndexDropdown.js

```diff
@@ -74,10 +78,57 @@ class IndexDropdown extends PureComponent {
         return this.placeholder.toLowerCase()
     }
 
+    renderTags() {
+        const tagOptions = this.props.tags.map((tag, i) => (
+            <span className={styles.tagPill}>
```

Comment:
Use the `key` prop when rendering a List. 
```suggestion
            <span key={i} className={styles.tagPill}>
```